_posts/2020-12-03-cloud-native-hpc.md

```diff
@@ -0,0 +1,120 @@
+Researcher then retrieves the output from the S3 bucket. 
+
+So there you have an example on how to run a compute intensive hpc workload in cloud, easy enough right? 
+
+### But what if we have huge amounts of data.. Petabytes of Data or more?
+Lets take for example Seismic Data or even what the **SKA** [Square Kilometre Array project](https://www.csiro.au/en/Research/Astronomy/ASKAP-and-the-Square-Kilometre-Array/SKA) will potentially generate.  The challenge comes when you try to move such large volumes of data around.. It can get expensive and complicated very quickly.  If you try to manage this data in an on-premise datacentre, you would ideally need a Hierarchical Storage Management (HSM) solution to help automate the lifecycle of your big data and integration with your application. Your data would live in the **ONLINE Storage Tier**  (high performance storage) when it’s needed for processing.  After processing has completed, the data can then be migrated to a **NEARLINE Storage Tier** (low cost storage) and then if long term Archiving of the data is needed, it would be migrated to the **OFFLINE/TAPE Storage Tier**.  
+
+<center><img src="/img/blog/hpc/hsm-storage.png" /></center><br/>
+
+
+Do you really want to maintain such a system yourself?  AWS have solved this problem by having FSx for Lustre integrate natively with Amazon S3, making it easy to access your S3 data to run data processing workloads.  FSx for Lustre transparently presents S3 objects as files and allows you to write results back to S3, which also means we now have access to a tiered storage solution with S3 lifecycle policies! **Viola!** Brilliant work AWS! Coming from a background where maintaining Lustre and HSM solutions for on-premise HPC is a 24x7 job equating to a high TCO, This is a huge win in my books...  Just a word of caution though.. Yes you now have automated your data management strategy with most of the responsibility shifting to AWS, but as we are now migrating your data to a cheaper storage tier in the cloud, please **DO** your homework and math on the costs to retrieve the data from the archive storage tiers along with potential egress charges. You have been warned **$$$**
+
+Another way to help reduce the volume of data being migrated to the cloud is to have a pre-processing step at the Edge - **Hybrid Cloud or Edge Computing**, which is when Kubernetes can be a good option to consider as it can cater for a wide variety of use cases.  This will help reduce your data footprint in the cloud, where you only transfer the pre-processed(cleansed, filtered, noise reduction) data for the core computational workloads in cloud rather than “all” the raw data. You can have the raw data written to tape or disk for archiving in a secure vault somewhere.
+
+
+### Cloud Supercomputing fighting the good fight!
```

Comment:
seems like it needs a hyphen or an `is`, e.g.

_Cloud Supercomputing - Fighting The Good Fight_

 or

_Cloud Supercomputing is Fighting The Good Fight_